src/cljdoc/server/routes.clj

```diff
@@ -39,6 +39,7 @@
 
 (defn utility-routes []
   #{["/jump/release/*project" :get nop :route-name :jump-to-project]
+    ["/redirect-to-version" :post nop :route-name :redirect-to-version]
```

Comment:
Really not sure about the naming of the route. And should it be placed under `/jump`?